docs/templates/plugins.rst.j2

```diff
@@ -61,30 +31,68 @@ Configuration
     Default: {% for default_item in actual_default %}``{{ default_item.pattern | default(default_item) }}``{% if not loop.last %}, {% endif %}{% endfor %}
             {% endif %}
         {% else %}
-            {% set _ = LOGGER.warn("%s/%s.%s: could not render default value, '%s'" | format(STEP, PLUGIN_ID, field_name, actual_default)) %}
+            {% set _ = LOGGER.warn("%s/%s.%s: could not render default value, '%s'" | format(STEP, plugin_id, field_name, actual_default)) %}
     Default: *could not render default value correctly*
         {% endif %}
     {% endif %}
 {%endmacro %}
 
-{% set ignored_fields = container_ignored_fields(PLUGIN_DATA_CLASS) %}
-{% set inherited_fields = container_inherited_fields(PLUGIN_DATA_CLASS) | sort %}
-{% set intrinsic_fields = container_intrinsic_fields(PLUGIN_DATA_CLASS) | sort %}
+.. _/plugins/{{ STEP }}:
 
-{% for field_name in intrinsic_fields %}
-{{ render_field(field_name) }}
-{% endfor %}
+{{ STEP | replace('-', ' ') | capitalize }} Plugins
+{{ '~' * (8 + (STEP | length)) }}
 
-{% if inherited_fields %}
-Common Keys
-"""""""""""
+{% for PLUGIN_ID, PLUGIN, PLUGIN_DATA_CLASS in PLUGINS() %}
+    {% if loop.first %}
+.. _plugins/{{ STEP }}/common-keys:
+
+Common keys
```

Comment:
```suggestion
Common Keys
```

so it better matches the documentation style. I see titles use Pascal Case

Reply:
Fixed in dad40531